### PR TITLE
feat: add feedback form on uninstall

### PR DESCRIPTION
**Pull request by secondl1ght**
### Describe the changes you have made in this PR

When a user uninstalls Alby they will now be directed to this URL: https://form.jotform.com/221745756722057

### Link this PR to an issue

#210 

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes (If any)

image

### How has this been tested?

Uninstalled the app and was taken to the feedback form.

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Review by escapedcat — APPROVED** [review]

**Comment by MoritzKa** [comment]
tested successfully on Chrome.
There is also a "remove from Chrome" button in the Chrome app store. I assume it triggers the same behaviour.

image

File: src/extension/background-script/index.ts [hunk 1]
```diff
@@ -156,3 +156,5 @@ init().then(() => {
     utils.openUrl("welcome.html");
   }
 });
+
+browser.runtime.setUninstallURL("https://form.jotform.com/221745756722057");
```
[the remark this answers is not in the document]
Comment (secondl1ght) [reply src/extension/background-script/index.ts:160]: Ok I can see if jotform has an embed option, if not I know Airtable does.

[the remark this answers is not in the document]
Comment (bumi) [reply src/extension/background-script/index.ts:160]: we can still redirect from there, but then it's a fixed URL in our control that is used in the extension. 
That URL in the extension we can not easily change once it is published

[the remark this answers is not in the document]
Comment (secondl1ght) [reply src/extension/background-script/index.ts:160]: Ohh ok, will do thanks.

**Comment by secondl1ght** [comment]
This PR needs to be merged at the same time: https://github.com/getAlby/website/pull/141/files

**Comment by escapedcat** [comment]
Converted to draft till the blocking PR is ready.

**Comment by secondl1ght** [comment]
This PR is ready to be merged when this one gets merged: https://github.com/getAlby/getalby.com/pull/182

**Comment by escapedcat** [comment]
💁‍♂️ https://github.com/getAlby/getalby.com/pull/182 has been merged

**Comment by secondl1ght** [comment]
> tipping_hand_man [getAlby/getalby.com#182](https://github.com/getAlby/getalby.com/pull/182) has been merged

Can we merge this then?

**Comment by escapedcat** [comment]
Let's merge it @bumi  
Just wondering why `uninstall` leads to `/thanks` :D

**Comment by secondl1ght** [comment]
> Let's merge it @bumi Just wondering why `uninstall` leads to `/thanks` :D

Haha I thought it would be nice for them to leave on a good note like 'thanks for trying out Alby, we hope you come back!'.

**Comment by MoritzKa** [comment]
builds were outdated and I cannot create new ones. 
If tested it some weeks ago and wanted to look into that 

 `uninstall` leads to `/thanks` 

uninstall should no lead to `/thanks` . We should thank users if they leave feedback.

**Comment by MoritzKa** [comment]
@escapedcat suggests ` /goodbye` . That´s a good suggestion. 

@secondl1ght can you adjust that on getalby.com/... ?

**Comment by escapedcat** [comment]
@secondl1ght do we need to update this as well or you already did?: https://github.com/getAlby/website/pull/141

**Comment by secondl1ght** [comment]
> @secondl1ght do we need to update this as well or you already did?: [getAlby/website#141](https://github.com/getAlby/website/pull/141)

Already updated it on the Rails repo here: https://github.com/getAlby/getalby.com/commit/8583e2981787ebaebf6394ef2bc33524a30938bb

So this is ready to be merged 🚀

**Pull request merged**